contrib/fiber.v2/example_test.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Please carefully read this https://github.com/DataDog/dd-trace-go/pull/786#discussion_r543170755 again 🙂

Reply:
i'm sorry, the path has been updated, I'm not sure what this is in reference to, might be good to highlight the target line at least.

Reply:
This means that the integration must be in the folder `contrib/gofiber/fiber.v2`

Reply:
oops, sorry, forgot the gofiber bit. Done now.